webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const Encore = require('@symfony/webpack-encore');
```

Comment:
This should be in `tests/Application`, not in the root of plugin, where it is useless.

Compare this with existing `webpack.config.js` in `tests/Application` and decide if we need some changes to adopt, or even if we want to replace old one with this one.